examples/netlify-functions/functions/getProducts/getProducts.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const { Product } = require('../../models');
+
+const handler = async(event) => {
+  try {
+    const Products = await Product.find();
+    return { statusCode: 200, body: Products };
```

Comment:
Make this lowercase `p` `products`. Also don't you need to do `body: JSON.stringify(products)`?